[DOCS] Updated language & code refs for Databricks Getting Started (#8751)
great-expectations/great_expectations: docs/docusaurus/docs/tutorials/getting_started/how_to_use_great_expectations_in_databricks.md

Because you used the DBFS for your Data Docs store, you need to download your Data Docs locally to view them. If you use a different store, you can host your data docs in a place where they can be accessed directly by your organization. 

> If you have the [Databricks CLI](https://docs.databricks.com/dev-tools/cli/index.html) installed and configured on your local machine, you can run the following commands from your local terminal to download your data docs and open the local copy of `index.html` to view your updated Data Docs:

Reviewer: ```suggestion
If you have the [Databricks CLI](https://docs.databricks.com/dev-tools/cli/index.html) installed and configured locally, run the following commands from your local terminal to download your data docs and open the local copy of `index.html` to view your updated Data Docs:
```

```bash
databricks fs cp -r dbfs:/great_expectations/uncommitted/data_docs/local_site/ great_expectations/uncommitted/data_docs/local_site/

cd great_expectations/uncommitted/data_docs/local_site

open -a "<YOUR_PREFERRED_BROWSER_NAME_HERE>" index.html
```

> The `displayHTML` command is another option for displaying Data Docs from within a Databricks notebook. There is a restriction, though, in that clicking a link in the displayed data documents returns an empty page. To view some validation results, use this method. For example:

Reviewer: ```suggestion
The `displayHTML` command is another option you can use to display Data Docs in a Databricks notebook. However, when you use this option, an empty page is returned when you click a link in the displayed data documents. To view validation results, use the following method:
```
